spec

```diff
@@ -188,37 +188,43 @@ endseg
 beginseg
     name "item_name_static"
     romalign 0x1000
-    include "build/baserom/item_name_static.o"
+    include "build/assets/textures/item_name_static/item_name_static.o"
+    number 10
 endseg
 
 beginseg
     name "map_name_static"
     romalign 0x1000
-    include "build/baserom/map_name_static.o"
+    include "build/assets/textures/map_name_static/map_name_static.o"
+    number 11
 endseg
 
 beginseg
     name "do_action_static"
     romalign 0x1000
-    include "build/baserom/do_action_static.o"
+    include "build/assets/textures/do_action_static/do_action_static.o"
+    number 07
```

Comment:
I don't think we include a leading zero in those anymore, so it should be cleaned up ideally (feel free to remove them from other numbers if there are any)
```suggestion
    number 7
```